app/gui/view/graph-editor/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 // === Features ===
 #![feature(associated_type_defaults)]
 #![feature(drain_filter)]
+#![feature(entry_insert)]
```

Comment:
I wonder why it was working before?

Reply:
I think it might have been almost stabilized and then reverted, but I did not confirm this yet. I remember removing these for one of the previous bumps.